libpod/container_internal.go

```diff
@@ -923,7 +923,7 @@ func (c *Container) cleanup() error {
 
 	// Clean up network namespace, if present
 	if err := c.cleanupNetwork(); err != nil {
-		lastError = nil
+		lastError = err
```

Comment:
This had been non-fatal since #302, but I don't see any discussion there about why it was non-fatal then.

Reply:
The only think I can think of was there is an issue where the cleanupNetwork can return an error that states the network does not exist, since it was cleaned up previously but there  is no current way for us to check for that error because the oci network code does not use constants.

Reply:
I think were were just being overly cautious there, given that we were already only warning on errors from `teardownNetwork()` in `cleanupNetwork()`.